__tests__/app.test.js

```diff
@@ -91,3 +91,27 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+
+  describe("/api/articles", () => {
+    it("GET responds with status code 200", () => {
+      return request(app).get("/api/articles").expect(200);
+    });
+    it("GET responds with slug and description fields", () => {
+      return request(app)
+        .get("/api/articles")
+        .then((response) => {
+            expect(response.body.articles.length).toBe(13)
+            response.body.articles.forEach((article) => {
+            expect(article).toMatchObject({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                body: expect.any(String),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
```

Comment:
Need a comment count 

comment_count, which is the total count of all the comments with this article_id. You should make use of queries to the database in order to achieve this.